client/transactions/list/style.scss

```diff
@@ -134,4 +134,64 @@ $gap-small: 12px;
 			height: auto;
 		}
 	}
+
+	.components-card__header {
+		.woocommerce-search.woocommerce-select-control
+			.woocommerce-select-control__listbox {
+			position: relative;
+			top: 5px;
+		}
+		.woocommerce-table__actions {
+			justify-content: space-between;
+
+			& > div {
+				width: 85%;
+				margin-right: 0;
+			}
+
+			button.woocommerce-table__download-button {
+				@include breakpoint( '<1040px' ) {
+					.woocommerce-table__download-button__label {
+						display: none;
+					}
+				}
+			}
+
+			.woocommerce-select-control.is-focused
+				.woocommerce-select-control__control {
+				flex-wrap: wrap;
+
+				.woocommerce-select-control__tags {
+					overflow-x: visible;
```

Comment:
💭 This line may not be required since there won't be overflow-x if the elements are wrapped (testing in Chrome).

Reply:
Nice catch, Thanks! Addressed via 135a410